frontend/src/app/utils/testResultCSV.test.ts

```diff
@@ -54,7 +54,11 @@ const data = [
       name: "Access Bio CareStart",
       manufacturer: "Access Bio, Inc.",
       model: "CareStart COVID-19 Antigen test*",
-      swabType: "258500001",
+      swabTypes: [
+        {
+          name: "Nasal swab",
+        },
+      ],
```

Comment:
Reasoning was also based on the query type - `GetFacilityResultsForCsvWithCount` currently only includes internal id and name for the swab types. Since internal id is just the UUID and doesn't refer to the specimen type code, name is the only field that makes sense to provide in the csv and so I updated the test data to reflect that the name field is text rather than a code. Interestingly it looks like the downloader had an existing bug where it wasn't populating device swab type because it wasn't referring to it correctly